weave/scorers/utils.py

```diff
@@ -65,14 +65,33 @@ def ensure_hf_imports() -> None:
         )
 
 
+def download_model_from_huggingface_hub(model_name: str) -> str:
+    """
+    Download a model from the Hugging Face Hub to a specified directory.
+
+    Args:
+        model_name (str): The name of the model on Hugging Face Hub.
+        local_dir (str or Path, optional): Directory to download the model to. Defaults to scorers_dir()/model_name.
+
+    Returns:
+        str: Path to the downloaded model directory.
+    """
+    from huggingface_hub import snapshot_download
+
+    model_dir_name = model_name.split("/")[-1].replace(":", "_")
+    local_dir = Path(scorers_dir()) / model_dir_name
+    return snapshot_download(model_name, local_dir=str(local_dir))
+
+
 def load_hf_model_weights(
-    model_name_or_path: str, default_model: Optional[str] = None
+    model_name_or_path: str = "", default_model: Optional[str] = None
 ) -> str:
     """Load the local model weights for a Hugging Face model.
 
-    If model_name_or_path is a directory, it is assumed to be the local model weights path.
-    If model_name_or_path is provided (non-empty), it is used to download the model using the existing download_model function.
-    If no model_name_or_path is provided, and a default_model is supplied, it downloads the default model.
+    If a default_model is supplied, it downloads the default model from the
+    Hugging Face Hub. Otherwise, if model_name_or_path is a directory, it is
+    assumed to be the local model weights path. If model_name_or_path is provided
+    and is not a local directory, it will be downloaded from wandb artifacts.
```

Comment:
Dont couple the implementations.  Write a third function that will do this dispatch